vectordraw.js

```diff
@@ -135,6 +135,48 @@ VectorDraw.prototype.createBoard = function() {
     this.board.on('up', this.onBoardUp.bind(this));
 };
 
+VectorDraw.prototype.renderPoint = function(idx, coords) {
+    var point = this.settings.points[idx];
+    var coords = coords || point.coords;
+    var board_object = this.board.elementsByName[point.name];
+    if (board_object) {
+        // If the point is already rendered, only update its coordinates.
+        board_object.setPosition(JXG.COORDS_BY_USER, coords);
+        return;
+    }
+    var opts = {
+        size: 1,
+        fixed: point.fixed,
+        name: point.name,
+        withLabel: false,
+        strokeColor: 'pink',
+        fillColor: 'pink',
+        showInfoBox: false
+    };
+    if (opts.fixed === undefined) {
```

Comment:
@smarnach global nit: I don't like that there're a lot of `something === undefined` checks. Maybe it would make sense to add a more intelligent "parse settings" step to initialization code instead of a `this.settings = _.extend(default_settings, settings);`?